src/docs/styles/components/_swatch.scss

```diff
@@ -260,32 +260,32 @@
   // gradients
   &-gradient-brand {
     @include swatch;
-    background: $gradient-brand;
+    @include linear-gradient($direction: '90deg', $style: $brand);
```

Comment:
thoughts on updating linter config to never?
https://github.com/stylelint-scss/stylelint-scss/blob/master/src/rules/at-mixin-named-arguments/README.md